packages/miniflare/src/plugins/kv/index.ts

```diff
@@ -71,18 +80,29 @@ export const KV_PLUGIN: Plugin<
 			bindings.push(...(await getSitesBindings(options)));
 		}
 
+		if (isWorkersWithAssets(options)) {
+			bindings.push(...(await getAssetsBindings(options)));
+		}
+
 		return bindings;
 	},
+
 	async getNodeBindings(options) {
 		const namespaces = namespaceKeys(options.kvNamespaces);
 		const bindings = Object.fromEntries(
 			namespaces.map((name) => [name, kProxyNodeBinding])
 		);
+
 		if (isWorkersSitesEnabled(options)) {
 			Object.assign(bindings, await getSitesNodeBindings(options));
 		}
+
+		if (isWorkersWithAssets(options)) {
```

Comment:
do we want this to be an `if...else if` situation instead, and allow declaring only one of `sitePath` or `assetsPath`?

Reply:
as per our sync meeting today, we should allow for both to work side-by-side. Will look into making that happen

Reply:
✅